wcosa/command/package_manager.py

```diff
@@ -0,0 +1,278 @@
+"""
+A lightweight package management system for WCosa.
+*_many functions operate on lists of data (common scenario, better efficiency).
+"""
+
+import json
+import os
+import re
+import sys
+
+import git
```

Comment:
Is the `git` package a new dependency? Then do add it to `setup.py` with an appropriate version

Reply:
Done